.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+Tracked by [Trello Card ID](https://trello.com/b/vQxfrlFD)
+
+---
+
+- [ ] The branch was rebased on the target branch,
+- [ ] Link to Trello card has been added,
+- [ ] PR has been assigned to a dev,
+- [ ] A review has been requested.
+
+---
+
+## 🎯 This PR...
```

Comment:
I left the emoji out, as I couldn't find it!
```suggestion
## 🎯 This PR aims at...
```

Reply:
I don't like the wording but feel free to use the 🎯-emoji in the backend template ! :)